### DISCUSSION TITLE
Drakvuf Web UI (Console) login credentials?

### DISCUSSION THREAD

**Discussion by ttownsend** [tklengyel/drakvuf#1761] [Q&A]
Hi all,

I've successfully installed Drakfuv, but I'm faced with a login screen at the web UI.
I didn't set any usernames of passwords during the install, so I don't know what credentials are required here.

What credentials are needed to access the web UI console, and how/where are these set?

Many thanks.

![Screenshot 2024-01-05 142720](https://github.com/tklengyel/drakvuf/assets/7128995/327eff00-eff0-455f-b5ce-b694ca7b35a1)

**Comment by tklengyel** [comment]
This is a question for https://github.com/CERT-Polska/drakvuf-sandbox

**Reply by ttownsend** [reply]
Thanks.  Already done.
